pkg/controller/nodeupdate/nodeupdate_controller.go

```diff
@@ -182,6 +182,7 @@ func (r *providerIDReconciler) getInstances (serviceInstance bluemixmodels.Servi
 	result = serviceInstanceResult{
 		instance: instance,
 	}
+	resultChan <- result
 	<-concurrencyController
```

Comment:
ShouldFix: Add this line (<-concurrencyController) in defer

Reply:
so you want to submit one more fix for this?

Reply:
I think, we can live with it for now, while pushing some changes later we can add this, I have commented here so that we will never forget this finding